pyEpiabm/pyEpiabm/sweep/intervention_sweep.py

```diff
@@ -46,7 +46,7 @@ def __call__(self, time):
         time : float
             Simulation time
         """
-        for intervention in self.interventions:
+        for intervention in self.interventions.keys():
             # TODO:
             # - Include an alternative way of case-count.
             #   Idealy this will be a global parameter that we can plot
```

Comment:
This is an old comment written by Isaac. I think he meant that we now need to sum over all the cells to get the number of cases and that it would be better to store somewhere the total number of cases instead of only per cell.